sdk/core/utils/redisclient/redisclient_test.go

```
@@ -0,0 +1,241 @@
+			}
+			_ = conn // held open, never responded to
+		}
+	}()
+
+	done := make(chan struct{})
+	go func() {
+		defer close(done)
+		// ReadTimeout set explicitly - the dial succeeds, it's the
+		// read-for-a-reply that hangs, and go-redis's default (5s) would
+		// otherwise bound that wait regardless of pingTimeout.
+		GetOrCreateRedisClient(&redis.Options{
```

golangci-lint (2.12.2)

[error] Error return value is not checked

(errcheck)